scss/_sticker.scss

```diff
@@ -2,6 +2,7 @@
   // scss-docs-start sticker-css-vars
   --#{$prefix}sticker-size: #{$sticker-size-md};
   --#{$prefix}sticker-font-weight: #{$font-weight-bold};
+  --#{$prefix}sticker-color: #{$black};
   --#{$prefix}sticker-background-color: #{$brand-orange};
```

Comment:
Since we are handling all the colors, would it be fine to introduce `$sticker-font-weight`, `$sticker-color` and `$sticker-bg-color` ?

Reply:
Yep I can do that, it won't change the CSS final size